sparse/usr/bin/droid/android-links.sh

```diff
@@ -1,14 +1,21 @@
 #!/usr/bin/env bash
 
-MNT=$(grep "Where=" /lib/systemd/system/android.mount | cut -d'=' -f 2)
+MNT=$(grep "Where=" /lib/systemd/system/android.mount | cut -d'=' -f 2)/media/0
+
+[[ "$(readlink -- /home/nemo/Pictures/Android)" == "/android/Pictures" ]] && unlink /home/nemo/Pictures/Android || echo "clean"
```

Comment:
Please unlink them all in one block, the code below is unreadable, whereas what we're trying to fix here is an unfortunate cleaning up after the error, should be separated from functionality